modules/rdma-creating-shared-device-rdma-ib.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/rdma-remote-direct-memory-access.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rdma-creating-shared-device-rdma-ib_{context}"]
+
+= Creating a shared device RDMA on Infiniband
+
+This procedure describes how to create the workload pods for a shared device RDMA for an Infiniband installation.
+
+.Procedure
+
+. Generate a custom pods resource, as shown below:
++
+[source,yaml]
+----
+$ cat <<EOF > rdma-ib-32-workload.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: rdma-ib-32-workload
+  namespace: default
+  annotations:
+    k8s.v1.cni.cncf.io/networks: example-ipoibnetwork
+spec:
+  nodeSelector: 
+    kubernetes.io/hostname: nvd-srv-32.nvidia.eng.rdu2.dc.redhat.com
+  serviceAccountName: rdma
+  containers:
+  - image: quay.io/redhat_emp1/ecosys-nvidia/gpu-operator:tools
+    name: rdma-ib-32-workload
+    command:
+      - sh
+      - -c
+      - sleep inf
+    securityContext:
+      privileged: true
+      capabilities:
+        add: [ "IPC_LOCK" ]
+    resources:
+      limits:
+        nvidia.com/gpu: 1
+        rdma/rdma_shared_device_ib: 1
+      requests:
+        nvidia.com/gpu: 1
+        rdma/rdma_shared_device_ib: 1
+EOF
+
+$ cat <<EOF > rdma-ib-32-workload.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: rdma-ib-33-workload
+  namespace: default
+  annotations:
+    k8s.v1.cni.cncf.io/networks: example-ipoibnetwork
+spec:
+  nodeSelector: 
+    kubernetes.io/hostname: nvd-srv-33.nvidia.eng.rdu2.dc.redhat.com
+  serviceAccountName: rdma
+  containers:
+  - image: quay.io/redhat_emp1/ecosys-nvidia/gpu-operator:tools
```

Comment:
Replace this quay with `- image: quay.io/edge-infrastructure/nvidia-tools:0.1.5`

Reply:
Replaced.